src/utils/frame.js

```diff
@@ -88,12 +112,64 @@ export function simplifyDisplayName(displayName) {
   return displayName;
 }
 
-export function formatDisplayName(frame: Frame) {
+export function formatDisplayName(
+  frame: Frame,
+  { shouldMapDisplayName = true } = {}
+) {
   const { displayName, library } = frame;
-  if (library) {
+  if (library && shouldMapDisplayName) {
     displayName = mapDisplayNames(frame, library);
   }
 
   displayName = simplifyDisplayName(displayName);
   return endTruncateStr(displayName, 25);
 }
+
+export function collapseFrames(frames) {
```

Comment:
yes this is ugly plane code, but it passes tests and we can fix it up another time :)

unless you can help me do better